scripts/aggregate-junit-xml.js

```diff
@@ -23,96 +23,160 @@
  * By default, all categories will be tested. To only process specific
  * categories, call the script with desired categories separated by a space.
  *
- * e.g. node aggregate-junit-xml.js integration vscode-integration
+ * e.g. 
+ * node aggregate-junit-xml.js -t integration vscode-integration
+ * npm run aggregateJUnit -- -t integration vscode-integration
```

Comment:
Can we add in this string that it will attempt to rerun tests for packages where no results were found one more time?